node/core/dispute-participation/src/lib.rs

```diff
@@ -119,7 +120,7 @@ where
 		match ctx.recv().await {
 			Err(_) => return,
 			Ok(FromOverseer::Signal(OverseerSignal::Conclude)) => {
-				tracing::info!(target: LOG_TARGET, "Received `Conclude` signal, exiting");
+				tracing::debug!(target: LOG_TARGET, "Received `Conclude` signal, exiting");
```

Comment:
And this?

Reply:
And many others in the same file

Reply:
It's easier (for now) to just display everything in Debug logs imo. I can change it back if you see it differently.

This PR is not intended to get merged at any point

Reply:
You can match logs with e.g. `level =~"DEBUG|INFO|ERROR|WARN"` ... just for reference.